endpoints-support.md

```diff
@@ -163,6 +163,13 @@
 |<sub>/rest/storage-systems/{id}/managedPorts/{portId}</sub>                              | GET      | :heavy_minus_sign:   | :heavy_minus_sign: | :heavy_minus_sign: | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:  |:heavy_minus_sign:    |
 |<sub>/rest/storage-systems/{id}/reachable-ports</sub>                                    | GET      | :white_check_mark:   | :white_check_mark: | :white_check_mark: | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/storage-systems/{id}/templates</sub>                                          | GET      | :white_check_mark:   | :white_check_mark: | :white_check_mark: | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|     **Storage Volume Attachments**                                                                                         |
+|<sub>/rest/storage-volume-attachments</sub>                                              | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |  :white_check_mark:  |
+|<sub>/rest/storage-volume-attachments/{id}</sub>                                         | GET      | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/storage-volume-attachments/repair</sub>                                       | GET      | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | ::white_check_mark:  |
+|<sub>/rest/storage-volume-attachments/repair</sub>                                       | POST     | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   | :white_check_mark:  |
+|<sub>/rest/storage-volume-attachments/{attachmentId}/paths</sub>                         | GET      | :white_check_mark:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | ::heavy_minus_sign:  |
```

Comment:
i don't see any support for these endpoints..so better to remove these from the file

Reply:
done